databases/week3/Part1_Movies_Queries.txt

```diff
@@ -0,0 +1,61 @@
+Part 1: Creating a database
+
+SET NAMES utf8mb4;
+
```

Comment:
The tables can not be created in this order
The movies has a FK from `certificate `